Assets/BossRoom/Scripts/Client/AnimationCallbacks/ClientAnimListener.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Client
+{
+    /// <summary>
+    /// Utility class used to listen for Animation-based events through Mecanim. This will invoke an Action when such
+    /// anim event is invoked.
+    /// </summary>
+    /// <remarks>
+    /// This class might in the future be refactored/removed with a Netcode for GameObjects refactor to NetworkAnimator.
```

Comment:
Has this been raised with the SDK team? Just to make sure this isn't lost in a comment here.

Reply:
Yup. It is https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/issues/1389.